x-pack/plugins/apm/server/routes/diagnostics/bundle/get_apm_events.ts

```diff
@@ -53,7 +55,7 @@ export async function getApmEvents({
     }),
     getEventWithMetricsetInterval({
       ...commonProps,
-      name: 'Metric: Service transaction (with summary field)',
+      name: 'Metric: Service transaction (8.7+)',
```

Comment:
@dgieselaar We talked about instead of naming the metrics based on the fields (with `transaction.duration.summary`) we should use the version where particular fields were introduced.